docs/core/policies.rst

```diff
@@ -62,10 +62,21 @@ when it isn't relevant and just respond with the same action no matter
 what happened before.
 
 You can alter this behaviour with the ``--augmentation`` flag.
-``--augmentation 0`` disables this behavior.
+Which allows you to set the ``augmentation_factor``.
+The ``augmentation_factor`` determines how many augmented stories are
+subsampled during training. Subsampling of the augmented stories is done in order to
+not get too many stories from augmentation, since their number
+can become very large quickly.
+The number of sampled stories is ``augmentation_factor``x10.
+
+``--augmentation 0`` disables all augmentation behavior.
+The :ref:`memoization_policy` is not affected by augmentation
```

Comment:
let's rather say all memoization based policies (we have 3 of them: memo, aug memo, form)